client/src/app/util/EmailVerification.js

```diff
@@ -0,0 +1,21 @@
+import emailjs, {init} from '@emailjs/browser';
+init('5A9OSqglcYcpKAMgd');
+
+export const VerifyEmail = (user) => {
+  const templateParams = {
+    to_email: user.useremail,
+    message: `http://localhost:3000/verify/${user.token}`,
```

Comment:
could you either add a comment here to remind us to make it dynamic, or edit it and merge?

Reply:
I added a comment to the pr for now while I figure out how to make the dynamic link work.

Reply:
The link is now dynamic